packages/common/utils/src/filter-characters.ts

```diff
@@ -0,0 +1,12 @@
+export function filterCharacters(text: string, validCharacters: string[] = []): string {
```

Comment:
```ts
type FilterPredicateFn = (value: T) => unknown;

export function filter<T extends string>(value: T, predicate: T[] | FilterPredicateFn<T>  = []): string {
    const list: string = Array.from(value);
    const filterFn: FilterPredicateFn = isFunctionLike(predicate) ? 
                                                                      predicate : 
                                                                      (value) => predicate.includes(value) || value === ' ' || value === '\\s';

   return list.filter(filterFn).join('');
}
```

Reply:
and move filter to `@angular-ru/common/string`

Reply:
changed